src/ansys/additive/core/examples/downloads.py

```diff
@@ -119,18 +123,18 @@ def download_small_wedge_slm_build_file():
 
 
 class MaterialTuningExampleInputFiles:
-    """Holds the example inputs for the material tuning process.
+    """Container for the example material tuning process inputs.
 
     Parameters
     ----------
     experiment_data_file : str
-        Path to the experiment data file.
+        Path to the experiment data file (CSV).
     material_parameters_file : str
-        Path to the material parameters file.
+        Path to the material parameters file (JSON).
```

Comment:
```suggestion
        Path to the material parameters file (JSON).
```
Don't we call this the material configuration file?

Reply:
Changed `material parameters file` to `material configuration  file`.